bin/common.sh

```diff
@@ -128,9 +127,70 @@ function for_each_cluster() {
   done
 }
 
-hf_templates_init() {
+function hf_templates_init() {
   local out_dir="$1"
   shift
-  [ -z "$*" ] && hf -f helmfile.tpl/helmfile-init.yaml template --skip-deps --output-dir="$out_dir" >/dev/null 2>&1
-  hf "$@" template --skip-deps --output-dir="$out_dir" >/dev/null 2>&1
+  [[ $all ]] && hf -f helmfile.tpl/helmfile-init.yaml template --skip-deps --output-dir="$out_dir" >/dev/null 2>&1
+  hf $(echo ${label:+"-l $label"} | xargs) template --skip-deps --output-dir="$out_dir" >/dev/null 2>&1
+}
+
+##############################################################################################
+# Use OPTIONS/LONGOPTS(LONGOPTIONS) to set additional parameters.
+# Please update this function comment if you add additional parameters.
+# Outputs:
+#     STDERR 2: parse_args without options
+#     STDERR 3: no '--' passed with options
+#     STDERR 4: special ERR reserved for checking if enhanced getopt is present on the host
+#     STDERR 5: no options specified
+# Returns:
+#     all -> if passed, sets to 'y' and can be used globally in conditional statements
+#     label -> if passed (e.g. label init=true), sets to label (e.g. 'init=true') and
+#              can be used globally in conditional statements
+# Resources:
+# - https://stackoverflow.com/a/29754866
+##############################################################################################
```

Comment:
This may look inconsistent but it is a function comment and I followed the style guide for that (https://github.com/google/styleguide/blob/gh-pages/shellguide.md#function-comments).